checkov/common/sca/output.py

```diff
@@ -129,6 +136,13 @@ def get_code_block(package: dict[str, Any], package_name: str, package_version:
     return [(0, f"{package_name}: {package_version}")]
 
 
+def get_fix_command_and_code(vulnerability_details: dict[str, Any],
+                             root_package_cve: dict[str, Any] | None = None) -> tuple[dict[str, Any] | None, str | None]:
```

Comment:
Does fixCommand is really a dict[str, Any] and not list[str]?

Reply:
yes, its dict